src/components/Navbar/Navbar.js

```diff
@@ -107,7 +108,9 @@ class Navbar extends Component {
                 minHeight: 280
               }}
             >
-            <Slideshow />
+
+              <Slideshow />
+              <TopVendors />
```

Comment:
Add this inside Switch in this way `<Route exact path="/" component={TopVendors} />`
Otherwise it will be visible on all pages